packages/storybook-test/stories/color-sample/color-sample.stories.tsx

```diff
@@ -36,10 +40,78 @@ type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
   name: 'Color Sample',
+  args: { color: '#ff1493' },
+  parameters: {
+    ariaLabelledBy: 'De kleur "#ff1493".',
+    docs: {
+      description: {
+        story: 'Een kleurstaal met beschrijving.',
+      },
+    },
+    status: { type: [] },
+  },
+  render(props, context) {
+    const id = useId();
+
+    return (
+      <>
+        <ColorSample {...props} aria-labelledby={id} />
+        <Paragraph id={id}>{context.parameters['ariaLabelledBy']}</Paragraph>
+      </>
+    );
+  },
+};
+
+export const SemiTransparentColorSample: Story = {
+  name: 'Color Sample (Semi-transparente kleur)',
+  args: { color: '#ff14937f' },
+  parameters: {
+    ariaLabelledBy: 'De semi-transparante kleur "#ff14937f".',
+    docs: {
+      description: {
+        story: 'Een kleurstaal met een semitransparante kleur en beschrijving.',
+      },
+    },
+    status: { type: [] },
+  },
+  render(props, context) {
+    const id = useId();
+
+    return (
+      <>
+        <ColorSample {...props} aria-labelledby={id} />
+        <Paragraph id={id}>{context.parameters['ariaLabelledBy']}</Paragraph>
+      </>
+    );
+  },
+};
+
+export const SoloColorSample: Story = {
+  name: 'Color Sample met title',
```

Comment:
Kan hier expliciet bij dat het om het SVG title-element gaat? Dan onstaat er geen verwarring met een title-attribuut. Dit doet ertoe in verband met toegankelijkheid: Een title-attribuut is vrijwel nooit een goed idee voor naamgeving.

Reply:
Aangepast